src/it/scala/org/zalando/spearheads/innkeeper/AcceptanceSpecsHelper.scala

```diff
@@ -29,7 +31,7 @@ object AcceptanceSpecsHelper extends ScalaFutures {
       .futureValue
   }
 
-  def postSlashRoutes(routeType: String)(token: String, routeName: String): HttpResponse = {
+  def postSlashRoutes(routeType: String)(routeName: String, token: String = ""): HttpResponse = {
```

Comment:
whoa... you have a fn, that takes two strings (in a static and strongly typed language) and you mix up the order of those two arguments? That is pretty evil...